src/extension/background-script/events/index.js

```diff
@@ -2,16 +2,18 @@ import PubSub from "pubsub-js";
 
 import { updateAllowance } from "./allowances";
 import {
-  paymentSuccessNotification,
-  paymentFailedNotification,
-  lnurlAuthSuccessNotification,
   lnurlAuthFailedNotification,
+  lnurlAuthSuccessNotification,
+  paymentFailedNotification,
+  paymentSuccessNotification,
 } from "./notifications";
 import { persistSuccessfullPayment } from "./persistPayments";
 
 const subscribe = () => {
   PubSub.subscribe("ln.sendPayment.success", paymentSuccessNotification);
   PubSub.subscribe("ln.sendPayment.failed", paymentFailedNotification);
+  PubSub.subscribe("ln.keysend.success", paymentSuccessNotification);
```

Comment:
should we have a for loop on ['sendPayment', 'keysend'] here? looks like there are now 3 subscriptions which are the same for both

Reply:
Makes sense! Done!